cpp/velox/substrait/SubstraitToVeloxPlanValidator.h

```diff
@@ -103,9 +106,17 @@ class SubstraitToVeloxPlanValidator {
 
   std::vector<std::string> validateLog_;
 
-  /// Used to get types from advanced extension and validate them.
-  bool validateInputTypes(const ::substrait::extensions::AdvancedExtension& extension, std::vector<TypePtr>& types);
+  /// Used to get types from advanced extension and validate them, then convert to a Velox type that has arbitrary
+  /// levels of nesting.
+  bool validateInputVeloxType(const ::substrait::extensions::AdvancedExtension& extension, TypePtr& out);
```

Comment:
`validateInputVeloxType` makes me feel that the input is velox type. Can we use some other name, e.g., `parseToVeloxType`?

Reply:
Changing to `parseVeloxType`. Thanks.